src/data/cognito.ts

```diff
@@ -46,3 +49,27 @@ export const listUsersInPool = async (
 
     return users;
 };
+
+const generateTemporaryPassword = (): string =>
+    Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
+
+export const addUserToPool = async (
+    cognito: CognitoIdentityServiceProvider,
+    userPoolId: string,
+    formUser: FormUser,
+): Promise<void> => {
+    const createUserParams: AdminCreateUserRequest = {
+        UserPoolId: userPoolId,
+        Username: formUser.email,
+        UserAttributes: [{ Name: 'custom:noc', Value: formUser.nocs }],
+        TemporaryPassword: generateTemporaryPassword(),
+    };
+    await cognito.adminCreateUser(createUserParams).promise();
+
+    const updateAttributesParams: AdminUpdateUserAttributesRequest = {
```

Comment:
I think we should be able to do this as part of the `adminCreateUser` call, we'll need to pass in the following to `UserAttributes` along with the custom:noc:

```
{ Name: 'email', Value: formUser.email },
{ Name: 'email_verified', Value: 'true' },
```